dotCMS/src/main/java/com/dotmarketing/util/LuceneQueryUtils.java

```diff
@@ -0,0 +1,70 @@
+package com.dotmarketing.util;
+
+import com.google.common.collect.ImmutableSet;
+import com.liferay.util.StringPool;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import org.apache.lucene.analysis.core.WhitespaceAnalyzer;
+import org.apache.lucene.index.Term;
+import org.apache.lucene.queryparser.classic.ParseException;
+import org.apache.lucene.queryparser.classic.QueryParser;
+import org.apache.lucene.search.BooleanClause;
+import org.apache.lucene.search.BooleanClause.Occur;
+import org.apache.lucene.search.BooleanQuery;
+import org.apache.lucene.search.TermQuery;
+
+
+public class LuceneQueryUtils {
+
+
+    /**
+     *
+     * @param luceneQuery
+     * @return
+     */
+    private static String removeQueryPrefix(final String luceneQuery){
+        final String cleanedUpQuery;
```

Comment:
you can use return (luceneQuery.startsWith("query_"))?luceneQuery.replace( "query_", StringPool.BLANK): luceneQuery;

it is just a style, up to you

Reply:
done.